diff --git a/dist/cli/commands/utils.js b/dist/cli/commands/utils.js
index 08dad800e24a9d8a7f22fc44ac19b1a2ad21204b..8bc5852d05198d9d04eff9d5517aa48968b5c8bf 100644
--- a/dist/cli/commands/utils.js
+++ b/dist/cli/commands/utils.js
@@ -7,6 +7,7 @@ import { printHeader, showWarningOnWindows } from "@opennextjs/aws/build/utils.j
 import logger from "@opennextjs/aws/logger.js";
 import { unstable_readConfig } from "wrangler";
 import { createOpenNextConfigIfNotExistent, ensureCloudflareConfig } from "../build/utils/index.js";
+import { pathToFileURL } from "url";
 export const nextAppDir = process.cwd();
 /**
  * Print headers and warnings for the CLI.
@@ -50,7 +51,8 @@ export async function retrieveCompiledConfig() {
         logger.error("Could not find compiled Open Next config, did you run the build command?");
         process.exit(1);
     }
-    const config = await import(configPath).then((mod) => mod.default);
+    // const config = await import(configPath).then((mod) => mod.default);
+    const config = await import(pathToFileURL(configPath).href).then((mod) => mod.default);
     ensureCloudflareConfig(config);
     return { config };
 }
